### PR TITLE
Add current phase Icon and background color for mobile process widget

### PR DESCRIPTION
**Pull request by djnunez-aot**
*Issue #992 :*
https://github.com/bcgov/met-public/issues/#992

-Add Icon for current phase

-Add background color for phases that are not in the current phase

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

File: met-web/src/components/engagement/view/widgets/PhasesWidget/PhasesWidgetMobile/EngagementPhaseMobile.tsx [hunk 1]
```diff
@@ -12,32 +14,42 @@ export const EngagementPhaseMobile = ({
...
+            backgroundColor={isCurrent ? backgroundColor : PAST_PHASE.backgroundColor}
             learnMoreBox={
-                <ReadMoreBox backgroundColor={accordionBackground} sx={{ p: 0, margin: 0 }}>
+                <ReadMoreBox
+                    backgroundColor={isCurrent ? accordionBackground : PAST_PHASE.backgroundColor}
+                    sx={{ p: 0, margin: 0 }}
+                >
                     <Grid
-                        sx={{ color: 'black' }}
+                        sx={{ color: isCurrent ? 'black' : 'white' }}
```
Comment (jadmsaadaot) [inline met-web/src/components/engagement/view/widgets/PhasesWidget/PhasesWidgetMobile/EngagementPhaseMobile.tsx:34]: Are your sure that the text color is black. Our primary text color is #494949. For this you can add the color: white conditionally. sx={ [ isCurrent && { color: 'white'} ]}

File: met-web/src/components/engagement/view/widgets/PhasesWidget/PhasesWidgetMobile/EngagementPhaseMobile.tsx [hunk 2]
```diff
@@ -12,32 +14,42 @@ export const EngagementPhaseMobile = ({
...
                         direction="row"
                         justifyContent="flex-start"
                         alignItems="flex-start"
                         spacing={1}
                     >
                         <Grid item xs={12}>
-                            <MetParagraph sx={{ fontWeight: 'bold', fontSize: '1.3rem', color: 'black' }}>
-                                {title}
-                            </MetParagraph>
+                            <MetParagraph sx={{ fontWeight: 'bold', fontSize: '1.3rem' }}>{title}</MetParagraph>
```
Comment (jadmsaadaot) [inline met-web/src/components/engagement/view/widgets/PhasesWidget/PhasesWidgetMobile/EngagementPhaseMobile.tsx:42]: weird that we are setting fontSize. This should be a standard by now. We have created MetHeaders and MetParagraph so we don't have to worry about setting fontSize anymore. What made us do it here?

**Review by jadmsaadaot — CHANGES_REQUESTED** [review]
We shouldn't use color black because our primary color isn't black and it's set in the theme. And we shouldn't set fontSize for typographies anymore.

File: met-web/src/components/engagement/view/widgets/PhasesWidget/PhasesWidgetMobile/EngagementPhaseMobile.tsx (hunk 2 above)
Comment (djnunez-aot) [reply met-web/src/components/engagement/view/widgets/PhasesWidget/PhasesWidgetMobile/EngagementPhaseMobile.tsx:42]: I think i was getting a unit test error saying a header tag cannot be within a div tag

File: met-web/src/components/engagement/view/widgets/PhasesWidget/PhasesWidgetMobile/EngagementPhaseMobile.tsx (hunk 1 above)
Comment (djnunez-aot) [reply met-web/src/components/engagement/view/widgets/PhasesWidget/PhasesWidgetMobile/EngagementPhaseMobile.tsx]: In Marie Anik's mobile design the text did not look like our primary color. Ill change it to the default though.

File: met-web/src/components/engagement/view/widgets/PhasesWidget/PhasesWidgetMobile/EngagementPhaseMobile.tsx (hunk 2 above)
Comment (djnunez-aot) [reply met-web/src/components/engagement/view/widgets/PhasesWidget/PhasesWidgetMobile/EngagementPhaseMobile.tsx]: I reverted it to MetHeader4 there was a jest bug before but I think it got patched.

**Review by tiagograf-AOT — APPROVED** [review]

**Review by jadmsaadaot — APPROVED** [review]

**Pull request merged**